### PR TITLE
[TRRFFDA-514] Create visually appealing radio lists for mobile view

### PR DESCRIPTION
**Pull request by ScriptSmith**

### CODE REVIEW

File: rdrf/rdrf/static/css/rdrf.css [hunk 1]
```diff
@@ -136,6 +136,18 @@ form .aligned ul.errorlist {
   margin-top: 5px;
 }
 
+/**************************************************************************
```
Comment (ScriptSmith) [inline rdrf/rdrf/static/css/rdrf.css]: Is there somewhere more appropriate to apply styling to the CDEs? Ideally, I'd prefer a webpack / react style approach, but that wouldn't work here.

File: rdrf/rdrf/forms/widgets/widgets.py [hunk 2]
```diff
@@ -322,22 +322,16 @@ def _get_value_range(self, cde_name):
...
 
     @staticmethod
     def usable_for_types():
         return {CommonDataElement.DATA_TYPE_RANGE}
 
-    def render(self, name, value, attrs=None, renderer=None):
-        html = super().render(name, value, attrs, renderer)
-        return self._transform(html)
-
-    def _transform(self, html):
```
Comment (ScriptSmith) [inline rdrf/rdrf/forms/widgets/widgets.py:336]: Many other CDEs use similar hacky methods to render the HTML, have a look at the rest of this file. Do you want me to look into fixing those using a similar approach to this one?

Comment (sztamas) [reply rdrf/rdrf/forms/widgets/widgets.py:336]: You can create a JIRA task for it, but let's not do large refactorings until we haven't done the merge work for Angelman yet.

File: rdrf/rdrf/static/css/rdrf.css (hunk 1 above)
Comment (sztamas) [reply rdrf/rdrf/static/css/rdrf.css]: Unfortunately, we're not in the webpack / react word yet, so until then this is the way to do it :-)

**Review by sztamas — APPROVED** [review]

**Pull request merged**